packages/grafana-data/src/types/config.ts

```diff
@@ -35,26 +36,6 @@ export enum GrafanaEdition {
   Enterprise = 'Enterprise',
 }
 
-/**
- * Describes available feature toggles in Grafana. These can be configured via the
- * `conf/custom.ini` to enable features under development or not yet available in
- * stable version.
- *
- * @public
- */
-export interface FeatureToggles {
```

Comment:
@mckn -- any thoughts on why/if this needs to bee in `@grafana/data`?  It seems like most of the config stuff really belongs in runtime